docs/tutorials/lattice-fermions.ipynb

```diff
@@ -43,9 +43,11 @@
   },
```

Comment:
<p>why are you transposing everything? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1622/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Two reasons. 1) It makes in my opinion the code easier to read, 2) it's faster than doing the opposite, because you want to slice the top-most dimension which is not contiguous, not the innermost dimension which is contiguous...</p><p> </p><p>We could even further speed this up by telling jax that the indices are sorted, but I won't do that here.</p><p> </p><p>The speedup is non negligible if you try to consider some systems with ~40 orbitals. Of course, the moment you add Backflow this becomes negligible again...</p>

Reply:
<p>Also with the code this way it is very simple to add hidden fermions. If you use your convention it is a bit annoying to get hidden fermions right)</p>